Migration.md

```diff
@@ -25,6 +25,9 @@ but with improved human-readability..
 - **sdf/Types.hh**: The `Inertia` class has been deprecated. Please use the
 ￼   `Inertial` class in the `gz-math` library.
 
+- `ignitionVersion()` is deprecated and will be removed in future versions.
+  Use `gzVersion()` instead.
```

Comment:
Once we remove the tick, we can also remove this note

Reply:
https://github.com/gazebosim/sdformat/pull/1040/commits/ac77e5ea225ce755a66357d1ed2bff935bd789f5 !